<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Trainer Registrieren</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ressources/icons/logo_smalltrans.png</normaloff>ressources/icons/logo_smalltrans.png</iconset>
  </property>
  <widget class="QFrame" name="formLogin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ressources/images/bg11.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="coachName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCoachTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Trainer Registrieren</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCoachName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Name*</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>541</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,35);
background-color: rgba(156, 139, 116,120);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="coachPhone">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCity">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Telefon*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coachAddress">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCoachCity">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>209</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Adresse*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coachZipCode">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>229</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCoachTelefon">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>209</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>PLZ*</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCoachBirthDate">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Gebutsdatum*</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCoachAnotherInfo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coachCity">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCoachEmail">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>209</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Stadt*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coachEmail">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QDateEdit" name="coachBirthDate">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit{
padding-left: 10px;
}</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2024</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ressources/icons/logo-onpadel.png.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCoachRegister">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 141, 0);
border-radius: 5px;
color:white;
}
QPushButton:hover{
background-color: rgb(246, 127, 47);
border: 2px solid rgb(255,255,255);
border-radius: 8px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Registrieren</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteCoach">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnDeleteCoach{
	color: rgb(255, 255, 255);
	border-radius:5px;
	background-color: rgb(254, 52, 1);
}
QPushButton#btnDeleteCoach:hover{
	border: 2px solid rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Löschen</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/recycle-bin.png</normaloff>ressources/icons/recycle-bin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="txtPflichtfelder">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>*Pflichtfelder</string>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>bg2</zorder>
   <zorder>coachName</zorder>
   <zorder>txtCoachTitle</zorder>
   <zorder>txtCoachName</zorder>
   <zorder>coachPhone</zorder>
   <zorder>txtCity</zorder>
   <zorder>coachAddress</zorder>
   <zorder>txtCoachCity</zorder>
   <zorder>coachZipCode</zorder>
   <zorder>txtCoachTelefon</zorder>
   <zorder>txtCoachBirthDate</zorder>
   <zorder>txtCoachAnotherInfo</zorder>
   <zorder>coachCity</zorder>
   <zorder>txtCoachEmail</zorder>
   <zorder>coachEmail</zorder>
   <zorder>coachBirthDate</zorder>
   <zorder>logo</zorder>
   <zorder>btnCoachRegister</zorder>
   <zorder>btnDeleteCoach</zorder>
   <zorder>txtPflichtfelder</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
